### ISSUE TITLE
Missing file numbers in compact mode?

### ISSUE THREAD

**Issue by CouchTomato87** [WandererXII/lishogi#624]
Not sure if this is intended or not, but file numbers disappear for compact mode.

**Comment by WandererXII** [comment]
The problem is they are hidden under the player/clock bar.
![image](https://user-images.githubusercontent.com/43043181/210905407-dd87d94b-88a5-4b41-a23c-eed8a6e77f25.png)
If you have coordinates set to edge or inside the board, they are there:
![image](https://user-images.githubusercontent.com/43043181/210905561-25b9ef56-64e4-489e-b986-61716069c030.png)

So the issues applies only when _Board coordinates_ are set to outside the board:
I tried making the files display inside the board even when set to outside the board, but it just looked weird with ranks still being outside the board.
I considered forcing both ranks and files to be inside the board, but that would make the option of _Board coordinates_ confusing and it affects only games (not analysis, puzzles, learn...) so some boards would have files inside when set to outside and some wouldn't - more confusion.
Positioning them above the bar, makes them overlap the top menu and I don't want to push the board even more lower. So I decided to just let it be and maybe figure out something better later... Open to ideas.

**Comment by ddugovic** [comment]
I'll offer a possibly controversial suggestion: the clock doesn't need to go on the player/clock bar (this compromise won't entirely fix the issue):

![image](https://user-images.githubusercontent.com/1834123/211140527-01454556-7ed5-436b-a178-56205c836354.png)

**Comment by WandererXII** [comment]
When I started working on compact mode, I had the clock there, it looked a bit out of place there so I moved them to the bar. But it might be worth considering and trying again.
This might also allow the clock font to be slightly larger, I kinda feel like the clock in compact mode is too small compared to the clock in the default mode, not sure if that's a problem or not.
But as you said this solves the issues only partially...
